test/integration/targets/setup_proxy/handlers/main.yml

```diff
@@ -0,0 +1,2 @@
+- name: stop proxy.py
+  command: kill {{ proxy_py_pid.content|b64decode }}
```

Comment:
Could you explain why is re-encoding of a.. number is needed? I'm a bit confused here. Oh, is it because it was produced by a slurp? Can we decode and put into a var right after reading the file? This piece of logic seems disconnected...

Reply:
It is because of slurp. Technically we could do a `set_fact` afterwards. Once we get the proejctions feature in, maybe for 2.18 we can do it as part of the register itself.